data/json/monsters.json

```diff
@@ -411,6 +411,49 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_gas_zombie",
```

Comment:
I suggest renaming it to `mon_zombie_gas`, as other zombies are named `mon_zombie_xxx`.